compare50/_data.py

```diff
@@ -112,17 +112,23 @@ class Submission:
     Represents a single submission. Submissions may either be single files or
     directories containing many files.
     """
-    _store = IdStore(key=lambda sub: (sub.path, sub.files))
+    _store = IdStore(key=lambda sub: (sub.path, sub.files, sub.large_files, sub.undecodable_files))
```

Comment:
Why include large_files and undecodable_files in this? There's no real reason to. Submissions are still uniquely identified by the path and files within them.

Reply:
There's a reason, check my first reply. In the case of single file submission, we'd still want to tell you which files compare50 did not include. However, now there could be several submissions with the same path and no files, but different excluded files. Compare50 would say they are equal, and throw one away once they enter a set. This actually happens if you for instance try to add distro files individually via something like "distro/*", some distro files might be ignored, but you would only ever see one

Reply:
oh u right my bad